e2e/utils/pages/homepage.page.ts

```diff
@@ -70,17 +71,6 @@ abstract class Homepage {
 		await this.#page.goto(HOMEPAGE_URL);
 	}
 
-	private async waitForModal({
-		modalOpenButtonTestId,
-		modalTestId
-	}: WaitForModalParams): Promise<Locator> {
-		await this.#page.getByTestId(modalOpenButtonTestId).click();
-		const modal = this.#page.getByTestId(modalTestId);
-		await modal.waitFor();
-
-		return modal;
-	}
-
```

Comment:
is this part related to this PR?

Reply:
Actually no, this change is only needed in my next PR. Removed.